acceleration/automatic_mixed_precision.ipynb

```diff
@@ -4,6 +4,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Copyright (c) MONAI Consortium  \n",
```

Comment:
Hi @dongyang0122 , thank you for helping the notebook update.

There are some format differences between MONAI tutorial and MONAI toolkit, and we plan to use scripts to convert notebooks from the tutorial to toolkit for packaging.

In other words, the tutorial has to follow the contributing guidelines so that we can make the conversion easy, and the notebook in the tutorial is going to be slightly different.

Regarding the copyright, please refer to [the guidelines](https://github.com/Project-MONAI/tutorials/blob/main/CONTRIBUTING.md#add-license) and copy-paste the markdown for the adjusted formats. 

Thanks in advance!

Reply:
Sure. I am following the guidelines to update the notebook.

Reply:
@mingxin-zheng, I have updated the copyright sections.